pkg/core/hash.go

```diff
@@ -63,6 +63,96 @@ func calculateConfigHash(children []configObject) (string, error) {
 	return fmt.Sprintf("%x", hashBytes), nil
 }
 
+// updateHashBank stores hashes of each child in the in memory hashbank. It is
+// not threadsafe, and not to be called other than by a helper function.  as the
+// children should be uniquely named within the namespace it does not store the
+// instance name.
+func (h *Handler) updateHashBank(children []configObject) error {
+	// Add the hashes of each childs data to the hashBank
+	for _, child := range children {
+		if child.object != nil {
+			switch child.object.(type) {
+			case *corev1.ConfigMap:
+				hash, err := hashData(getConfigMapData(child))
+				if err != nil {
+					// log event and continue?
```

Comment:
@mthssdrbrg I'm unsure of how to handle errors here. Should a single failure cause the entire thing to bail and re-reconcile? Do we want to log and silently fail?